Assignment2/DanielMiM-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,46 @@
+{
+    "@context": {
+        "inf": "http://example.org",
+        "asign": "http://example.org",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "name": "xsd:string",
+        "hometown": "xsd:string",
+        "age": "xsd:integer",
+        "almaMater": "xsd:string",
+        "parents": "xsd:string",
+        "teachers":"xsd:string",
+        "description":"xsd:string"
```

Comment:
Syntactically is ok, but you have to define each attribute with a specific vocabulary concept/property name because when you want to retrieve, for example your age, if you query "xsd:string" you will not be able to distinguish what data you ask for. If you have your own vocabulary you can use it to define your attributes e.g. "name":"yourvocab:name", "age":"yourvocab:age", etc.